### ISSUE TITLE
MethodError: no method matching compilecache_path

### ISSUE THREAD

**Issue by aviks** [ianshmean/SystemBenchmark.jl#45]
```
julia> using SystemBenchmark

julia> comp = comparetoref()
[ Info: CUDA.functional() == false. No usable GPU detected
Compilation tests 81%|███████████████████████████████████████████████████████▉             |  ETA: 0:00:40
ERROR: MethodError: no method matching compilecache_path(::Base.PkgId)
Closest candidates are:
  compilecache_path(::Base.PkgId, ::UInt64) at loading.jl:1272
Stacktrace:
 [1] compilecache_init(pkg::Base.PkgId)
   @ SystemBenchmark ~/data/.julia/packages/SystemBenchmark/vhsLA/src/SystemBenchmark.jl:253
 [2] runbenchmark(; printsysinfo::Bool, slowgcsleep::Float64)
   @ SystemBenchmark ~/data/.julia/packages/SystemBenchmark/vhsLA/src/SystemBenchmark.jl:175
 [3] comparetoref(; slowgcsleep::Float64)
   @ SystemBenchmark ~/data/.julia/packages/SystemBenchmark/vhsLA/src/SystemBenchmark.jl:93
 [4] comparetoref()
   @ SystemBenchmark ~/data/.julia/packages/SystemBenchmark/vhsLA/src/SystemBenchmark.jl:93
 [5] top-level scope
   @ REPL[7]:1
```

```
julia> versioninfo()
Julia Version 1.6.1
Commit 6aaedecc44 (2021-04-23 05:59 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) Platinum 8259CL CPU @ 2.50GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, skylake-avx512)
```

**Comment by aviks** [comment]
duplicate of #34

**Comment by IanButterworth** [comment]
Fixed as of v0.4
